code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -22,6 +22,7 @@
 	head_icon = 'icons/mob/pai_item_head.dmi'
 	radio = /obj/item/radio/headset/silicon/pai
 	can_buckle_to = FALSE
+	mobility_flags = MOBILITY_FLAGS_DEFAULT | MOBILITY_REST
```

Comment:
should probably mention that pAIs can still rest too

someday, we should add resting for Ian, then give mulebots the ability to run him (and Runtime) over (like they do with prone humans) if he's resting

Reply:
I copypasted the title and forgot to update it, sorry!